contrib/horizon/blazardashboard/api/blazar.py

```diff
@@ -160,3 +178,28 @@ def reservation_calendar(request):
     host_reservations = dictfetchall(cursor)
 
     return host_reservations
+
+def available_nodetypes():
+    cursor = connections['blazar'].cursor()
+    sql = '''\
+    SELECT DISTINCT
+        capability_value
+    FROM
+        computehost_extra_capabilities
+    WHERE
+        capability_name = 'node_type'
+        AND deleted = '0'
+    GROUP BY
+        computehost_id
```

Comment:
Vestigial from when I was finding the current node_type by looking at `MAX(created_at)` rather than everything that isn't deleted. Removed.